index.js

```diff
@@ -1,12 +1,44 @@
 /* eslint-disable global-require */
 
+import PushNotificationIOS from "@react-native-community/push-notification-ios";
 import React from 'react';
 import { AppRegistry, Platform, UIManager } from 'react-native';
+import PushNotification from "react-native-push-notification";
 import { enableScreens } from 'react-native-screens';
 import { name as appName } from './app.json';
 import { config } from './src/config';
 import { App } from './src/domains/core/App';
 
+PushNotification.configure({
```

Comment:
This logic probably belongs within a hook or service call, invoked from `App.tsx`

Reply:
I agree about the hook/service call. 

As for where this should be called, the documentation says to not call it inside a component or App. https://www.npmjs.com/package/react-native-push-notification#usage

Reply:
Eek, yea we're gonna be dealing with some race conditions around this & the rest of the app init, fun